internal/cephes/zeta.go

```diff
@@ -1,81 +1,53 @@
-// Copyright ©2016 The gonum Authors. All rights reserved.
+// Derived from SciPy's special/cephes/zeta.c
+// https://github.com/scipy/scipy/blob/master/scipy/special/cephes/zeta.c
+
```

Comment:
Maybe add in here something that mentions that Stephen L. Moshier has made the original source free to use. The text is in the README in the cephes package (what I quoted [here](https://github.com/gonum/mathext/pull/24#issuecomment-281573279)). I would suggest something like: "Made freely available by Stephen L. Moshier without support or guarantee." or similar just below the derivation claim above.